src/SFML/Graphics/Shader.cpp

```diff
@@ -250,6 +250,38 @@ Shader::~Shader()
         glCheck(GLEXT_glDeleteObject(castToGlHandle(m_shaderProgram)));
 }
 
+////////////////////////////////////////////////////////////
+Shader::Shader(Shader&& source) noexcept :
+m_shaderProgram(std::exchange(source.m_shaderProgram, 0U)),
+m_currentTexture(std::exchange(source.m_currentTexture, 0)),
```

Comment:
-1 is the default value so it probably makes sense to assign -1 to the moved-from texture index.
```suggestion
m_currentTexture(std::exchange(source.m_currentTexture, -1)),
```

Reply:
Good catch, my bad!